app/views/pages/team.html.haml

```diff
@@ -17,6 +17,16 @@
               universities to bring hackathons to colleges all around the globe.
               Dave enjoys studying hybrid online and offline communities.
           .small-12.medium-6.columns.profile
+            = image_tag 'team/harrison_shoebridge.jpg', class: 'profile-img'
+            .heading
+              %h1 Harrison Shoebridge
+              %h2 Intern
+            %p.bio
+              Harrison joined Hack Club officially in December 2015, previously
```

Comment:
@MaxWofford The date occurs in Dave's, Jonathan and Zachs as well. Should we change that too?